msp/msp_exp_handler.c

```diff
@@ -626,11 +626,17 @@ void msp_exprecv_complete(unsigned char opcode)
 			break;
 
 		case MSP_OP_APPLY_CUBES_CITI_CONF:
```

Comment:
I guess you forgot to change the name of the macro, should it be `MSP_OP_SELECT_NVM_CITI_CONF`?